### DISCUSSION TITLE
Question about options in model

### DISCUSSION THREAD

**Discussion by anirbanroy16** [pybamm-team/PyBaMM#3427] [Q&A] [answered]
Could anybody kindly throw a bit more light about what these options mean? Any pointers to requisite literature would be an icing on the cake.

“current collector”str
Sets the current collector model to use. Can be “uniform” (default), “potential pair” or “potential pair quite conductive”.

“diffusivity”str
Sets the model for the diffusivity. Can be “single” (default) or “current sigmoid”. A 2-tuple can be provided for different behaviour in negative and positive electrodes.

“electrolyte conductivity”str
Can be “default” (default), “full”, “leading order”, “composite” or “integrated”.

“exchange-current density”str
Sets the model for the exchange-current density. Can be “single” (default) or “current sigmoid”. A 2-tuple can be provided for different behaviour in negative and positive electrodes.

**Comment by valentinsulzer** [comment, marked as answer]
"current collector" - see this paper https://arxiv.org/pdf/2008.03691.pdf
"diffusivitiy" and "exchange-current density" - if using "current sigmoid" different parameters are used for charge and discharge (using a sigmoid function of the current to smoothly find its sign)
"electrolyte conductivity" - see https://www.sciencedirect.com/science/article/pii/S0013468621008148

**Reply by anirbanroy16** [reply]
Thank you very much! What about this one?

“SEI”str
Set the SEI submodel to be used. Options are:

“reaction limited”, “reaction limited (asymmetric)”, “solvent-diffusion limited”, “electron-migration limited”, “interstitial-diffusion limited”, “ec reaction limited” or “ec reaction limited (asymmetric)”: [pybamm.sei.SEIGrowth](https://docs.pybamm.org/en/latest/source/api/models/submodels/interface/sei/sei_growth.html#pybamm.sei.SEIGrowth)

**Reply by valentinsulzer** [reply]
See the SEI submodel class for references about specific SEI models

**Comment by anirbanroy16** [comment]
and this?

“surface form”str
Whether to use the surface formulation of the problem. Can be “false” (default), “differential” or “algebraic”.

**Reply by anirbanroy16** [reply]
Also, I thought SPM treats electrode as equivalent of one particle. How does this translate to x-axis?

“x-average side reactions”: str
Whether to average the side reactions (SEI growth, lithium plating and the respective porosity change) over the x-axis in Single Particle Models, can be “false” or “true”. Default is “false” for SPMe and “true” for SPM.
